src/algebra/algebra/basic.lean

```diff
@@ -518,6 +518,10 @@ lemma map_finsupp_sum {α : Type*} [has_zero α] {ι : Type*} (f : ι →₀ α)
 @[simp] lemma map_bit1 (x) : φ (bit1 x) = bit1 (φ x) :=
 φ.to_ring_hom.map_bit1 x
 
+-- TODO[gh-6025]: make this an instance once safe to do so
+lemma subsingleton [subsingleton A] : subsingleton (A →ₐ[R] B) :=
+⟨λ f g, alg_hom.ext $ λ x, subsingleton.elim 0 x ▸ f.map_zero.trans g.map_zero.symm⟩
+
```

Comment:
Oops. we already have this but it ended up being called `subalgebra.alg_hom.subsingleton`

Reply:
Opened #8140 to fix the names.